ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/management-api-models.j2

```diff
@@ -0,0 +1,27 @@
+{# Management API Models #}
+{% if management_api_models is arista.avd.defined %}
+
+## Management API Models
+
+### Management API Models Summary
+
+| Provider | Path | Enabled |
+| -------- | ---- | ------- |
+{%     if management_api_models.providers is arista.avd.defined %}
+{%         for provider in management_api_models.providers | arista.avd.convert_dicts('name') | arista.avd.natural_sort('name') %}
+{%             if provider.paths is arista.avd.defined %}
+{%                 for path in provider.paths | arista.avd.convert_dicts('path') | arista.avd.natural_sort('path') %}
+{%                     set enabled = path.enabled | arista.avd.default(true) %}
+| {{ provider.name }} | {{ path.path }} | {{ enabled }} |
+{%                 endfor %}
+{%             endif %}
+{%         endfor %}
+{%     endif %}
+
+### Management API Models Configuration
+
+```eos
+{% include 'eos/management-api-models.j2'%}
+```
+
+{% endif %}
```

Comment:
Required a blank line without spaces at the end of file

Reply:
fixed